mne/epochs.py

```diff
@@ -355,7 +355,7 @@ class BaseEpochs(ProjMixin, ContainsMixin, UpdateChannelsMixin, ShiftTimeMixin,
         Iterable of indices of selected epochs. If ``None``, will be
         automatically generated, corresponding to all non-zero events.
     drop_log : list | None
-        List of lists of strings indicating which epochs have been marked to be
+        List of tuple of strings indicating which epochs have been marked to be
```

Comment:
Is there a reason that `BaseEpochs` here wants a *list* of tuples, but the `Epochs` constructor wants a *tuple* of tuples?  Since users are discouraged from using `BaseEpochs` for anything but type checking, it's probably not likely to cause problems... just curious if there's a rationale.

Reply:
Just missed updating this docstring